skills.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Skills
+---
+
+{%- assign langs_framework_misc = "langs,frameworks,misc" | split: "," -%}
+{%- assign self_want = "self,want" | split: "," -%}
+
+{% for stype in langs_framework_misc -%}
+
+<h1> {{ site.data.skillset["titles"][stype] }} </h1>
+
+    {%- for skill_ in site.data.skillset[stype] -%}
+        {%- assign skill_name = skill_[0] -%} {%- comment -%} skill name we are searching for {%- endcomment -%}
+        {%- assign skill = skill_[1] -%} {%- comment -%} possible skill-related skills {%- endcomment -%}
+
+        {%- comment -%} Search if there is at least an author holding skills for this skill
+                        and continues to next skill if there is noone {%- endcomment -%}
```

Comment:
s/noone/none

Reply:
I actually intended noone, like "no-one", synonymous of nobody :)

Reply:
[Then s/noone/no one](https://www.grammarly.com/blog/no-one-noone/) (sorry, I'm compensating for my lack of Liquid knowledge with English)

Reply:
Woah, _the more you know!_

I'll fix that with some other typos I've already spotted, ty!